core/templates/domain/statistics/ExplorationStatsObjectFactory.ts

```diff
@@ -77,13 +75,12 @@ export class ExplorationStatsObjectFactory {
 
   createFromBackendDict(
       backendDict: IExplorationStatsBackendDict): ExplorationStats {
-    const stateStatsMapping = new Map(
-      Object.entries(backendDict.state_stats_mapping).map(
-        ([stateName, stateStatsBackendDict]) => [
+    const stateStatsMapping = (
+      new Map(Object.entries(backendDict.state_stats_mapping).map(
+        ([stateName, stateStatsDict]) => [
```

Comment:
I suggest you keep the name as it was, since stateStatsDict sounds like a dict containing stateStats frontend objects.